upsert.gemspec

```diff
@@ -32,7 +32,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency 'rake', '~>10.1.1'
 
   if RUBY_VERSION >= '1.9'
-    gem.add_development_dependency 'activerecord-import'
+    gem.add_development_dependency 'activerecord-import', '0.3.1'
```

Comment:
I'll have to review why I did that.  Something with the import not running properly, I believe.  That might have been the version that was used implicitly prior and I just pinned it to make sure it wasn't updated.